civicrm.module

```diff
@@ -172,15 +172,19 @@ function civicrm_menu_alter(&$items) {
 /**
  * Implements hook_layout_access_info().
  *
- * Adding a CiviCRM checksum visibility condition.
+ * Adding a CiviCRM checksum visibility condition and a URL condition that
+ * considers the query parameters in the URL.
  */
 function civicrm_layout_access_info() {
-  $info = array(
-    'civicrm_checksum' => array(
-      'title' => t('CiviCRM checksum'),
-      'description' => t('Has the page been loaded with a valid CiviCRM checksum?'),
-      'class' => 'CivicrmChecksumLayoutAccess',
-    ),
+  $info['civicrm_checksum'] = array(
+    'title' => t('CiviCRM checksum'),
+    'description' => t('Has the page been loaded with a valid CiviCRM checksum?'),
+    'class' => 'CivicrmChecksumLayoutAccess',
+  );
+  $info['civicrm_urls'] = array(
```

Comment:
@bugfolder 
If memory serves, recent versions of PHP prefer arrays to be declared before using them. This could just be added as a new key in the array that is declared in the original code.

Reply:
All versions of PHP allow this. From [the PHP manual](https://www.php.net/manual/en/language.types.array.php), 
```
$arr[key] = value;
$arr[] = value;
// key may be an int or string
// value may be any value of any type
```
> If `$arr` doesn't exist yet, it will be created, so this is also an alternative way to create an array.

Reply:
Continuing that quote: 

> ...This practice is however discouraged... 

I don't think the CiviCRM or Backdrop coding standards specifically have a section about this but I used the language "prefer" originally. I'll leave it to @herbdool as maintainer on his preferences here.

Reply:
I think this is fine.